elements/pf-progress-stepper/test/pf-progress-steps.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import { test } from '@playwright/test';
+import { PfeDemoPage } from '@patternfly/pfe-tools/test/playwright/PfeDemoPage.js';
+
+const tagName = 'pf-progress-steps';
```

Comment:
`'pf-progress-steps'` or `'pf-progress-stepper'`?

Reply:
It would actually be `pf-progress-step` if the e2e test is targeting the sub-component, but I don't think that is the intent. 

I would think it is meant to target the parent component, so in that case, both tests are named incorrectly `pf-progress-steps.spec.ts` and `pf-progress-steps.e2e.ts` should both be `pf-progress-stepper.*` to follow the file naming convention.